2-mandatory/3-financial-times.js

```diff
@@ -5,16 +5,24 @@
     Implement the function below, which will return a new array containing only article titles which will fit.
 */
 function potentialHeadlines(allArticleTitles) {
-    // TODO
+  let shortArticles = allArticleTitles.filter(title => title.length < 66); 
+  return shortArticles;
 }
 
 /*
     The editor of the FT likes short headlines with only a few words!
     Implement the function below, which returns the title with the fewest words.
     (you can assume words will always be seperated by a space)
 */
+
 function titleWithFewestWords(allArticleTitles) {
-    // TODO
+  let arrOfLength = [];
+  for (let i = 0; i < allArticleTitles.length; i++) {
+    arrOfLength.push(allArticleTitles[i].split(" ").length - 1);
```

Comment:
Why did you use `.push()` here instead of `arrOfLength[i]`?

Naming variables is hard. What data is being stored in the `arrOfLength` variable? Can you think of a better name?